i.sentinel_2.sen2cor/i.sentinel_2.sen2cor.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python3
+#
+############################################################################
+#
+# MODULE:      i.sentinel_2.sen2cor
+# AUTHOR(S):   Guido Riembauer, <riembauer at mundialis.de>
+#
+# PURPOSE:     Runs atmospheric correction on a single Sentinel-2 L1C scene using sen2cor
+# COPYRIGHT:   (C) 2021-2023 by Guido Riembauer, mundialis GmbH & Co. KG and the GRASS Development Team
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#############################################################################
+
+# %Module
+# % description: Runs atmospheric correction on a single Sentinel-2 L1C scene using sen2cor.
+# % keyword: imagery
+# % keyword: atmospheric correction
+# % keyword: radiometric conversion
+# % keyword: radiance
+# % keyword: reflectance
+# % keyword: satellite
+# % keyword: Sentinel
+# % keyword: atmosphere
```

Comment:
```suggestion
# % keyword: imagery
# % keyword: satellite
# % keyword: Sentinel
# % keyword: atmospheric correction
# % keyword: radiometric conversion
# % keyword: radiance
# % keyword: reflectance
# % keyword: atmosphere
```

Following second keyword as in (similiar) https://grass.osgeo.org/grass-stable/manuals/addons/i.sentinel.preproc.html